user_stories.md

```diff
@@ -0,0 +1,27 @@
+As a marketing research team member, I:
+
+1 Want customers to only see items they are most likely to purchase
+2 Want to know how many of each product sells
+3 Want to know how many times each product has been viewed 
+4 Want a comparison of views to clicks/purchases
+5 Does not want user to see any of the data until they are done
+
+
+
+As a developer research team member, I:
+
+1 Want to make a Constructor to add items to a catalog
+2 Want to cycle through 3 images each iteration that'll appear side-by-side-by-side
+3 Want to limit User to a finite number of iterations
+4 Want to force the user to select an image in order to move onto the next iteration
+5 Want to provide MRT with data that's collected from each iteration
+
+
+
+As a Focus Group Participant, I:
+
+1 Do not want to see any immediate repeat selections
+2 Want to have a good selection of choices
+3 Want to have an exciting and consistent layout 
+4 Want to know that my choices count
+5 
```

Comment:
All of your stories need a "So That ..." clause to give them more depth and clarity.